pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.poetry]
+name = "django-task-app"
+version = "0.1.0"
+description = "Andrei's Django Wellence Task"
+authors = ["Andrei"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.12"
+Django = "^5.1.1"
+django-compressor = "^4.5.1"
+django-ninja = "^1.3.0"
```

Comment:
This could have been added in the follow-up PR when adding the REST API, but this works too